texture-font.c

```diff
@@ -381,28 +393,54 @@ texture_font_load_face( texture_font_t *self, float size )
 	    }
 	    break;
 	}
-    }
 
-    /* Select charmap */
-    error = FT_Select_Charmap(self->face, FT_ENCODING_UNICODE);
-    if(error) {
-        freetype_error( error, "FT_Error (%s:%d, code 0x%02x) : %s\n",
-			__FILENAME__, __LINE__, FT_Errors[error].code, FT_Errors[error].message);
-        goto cleanup_face;
-    }
+	/* Select charmap */
+	error = FT_Select_Charmap(self->face, FT_ENCODING_UNICODE);
+	if(error) {
+	    freetype_error( error, "FT_Error (%s:%d, code 0x%02x) : %s\n",
+			    __FILENAME__, __LINE__, FT_Errors[error].code, FT_Errors[error].message);
+	    goto cleanup_face;
+	}
 
-    /* Set char size */
-    error = FT_Set_Char_Size(self->face, (int)(size * HRES), 0, DPI * HRES, DPI);
-    
-    if(error) {
-        freetype_error( error, "FT_Error (%s:%d, code 0x%02x) : %s\n",
-			__FILENAME__, __LINE__, FT_Errors[error].code, FT_Errors[error].message);
-        goto cleanup_face;
+	if( texture_is_color_font( self ) ) {
```

Comment:
texture_is_color_font doesn't work for the Apple Emoji font. Here's what works for me:

```suggestion
      if( FT_HAS_FIXED_SIZES( self->face ) ) {
```

Reply:
Note: I have added this change and the one above to my PR to your repo